test/README.md

```diff
@@ -281,7 +281,21 @@ disabled.spec.<spec> (e.g. disabled.spec.linux_x86-64)
     or were skipped, along with other information like execution time and
     error messages, exceptions and logs from the individual test methods.
 
+  * tap result files
```

Comment:
Please change to TAP (Test Anything Protocol) result files  (and change any subsequent use of tap to TAP as its an acronym).